etc/frontend/events.xml

```diff
@@ -0,0 +1,6 @@
+<?xml version="1.0"?>
+<config xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:noNamespaceSchemaLocation="urn:magento:framework:Event/etc/events.xsd">
+    <event name="customer_register_success">
+        <observer name="InchooAddDefaultBookmarkList" instance="Inchoo\ProductBookmark\Observer\AddDefaultBookmarkList" />
```

Comment:
nepraktičan način dodavanja liste:

default lista se napravi tek kada se ulogiraš i odeš vidjeti liste, ako se ulogiraš i odeš na product page prije toga, bit će prazan dropdown i ništa se neće moći dodati, customeri bi zvali support vjerojatno. Nije baš praktično. Ja bi napravio default liste za sve usere skriptom ako će se već tako insertati u bazu i onda bi bile vidljive bilo gdje ili našao neki drugi način, npr stavio default lista da je ID 0, i stavio ju u isto tablicu gdje je customer_id pa po tome svrstavao. npr jedna tablica list_id, customer_id, product_id, website_id, a druga da sadrži imena lista samo

Reply:
Maknio observer, ako user nema listu - moze bookmarkati i napravi se default lista.